test/distributed/test_c10d_common.py

```diff
@@ -1236,6 +1237,34 @@ def recv(self, tensor_list, src, tag=0):
         return DummyWork()
 
 
+class CompilerTest(MultiProcessTestCase):
+    def setUp(self):
+        super(CompilerTest, self).setUp()
+        self._spawn_processes()
+
+    def tearDown(self):
+        super(CompilerTest, self).tearDown()
+        try:
+            os.remove(self.file_name)
+        except OSError:
+            pass
+
+    def test_make_fx(self):
+        os.environ['MASTER_ADDR'] = 'localhost'
+        os.environ['MASTER_PORT'] = '6789'
+        dist.init_process_group("gloo", rank=self.rank, world_size=self.world_size)
```

Comment:
You must destroy the PG at the end of your test otherwise other tests won't be able to do the same.
You can use a wrapper like this one that simplifies this task:
https://github.com/pytorch/pytorch/blob/master/torch/testing/_internal/distributed/_shard/sharded_tensor/__init__.py#L81


You need to filter out this test if gloo is not avaible. The way we do it is with a `requires_gloo` wrapper:
https://github.com/pytorch/pytorch/blob/master/torch/testing/_internal/common_distributed.py#L256